packages/integration-sdk-cli/src/bocchi/utils/types.ts

```diff
@@ -86,7 +92,7 @@ export interface Step {
   mappedRelationships?: {
     _class: RelationshipClass;
     direction: 'FORWARD' | 'REVERSE';
-    fieldMappings:
+    fieldMappings?:
```

Comment:
Is there a reason why a mapped relationship wouldn't be expected to have mappings?